### PR TITLE
feat(accountDelete): lambda to hit queueDelete endpoint

### PR DESCRIPTION
**Pull request by kschelonka**
## Goal
Processes ACCOUNT_DELETE events from eventbridge,
and enqueues batches for deleting a user's list

~Wait for #306 to be merged to ensure the schemas are correct.~ **DONE**

**NOTE:** Tested on dev

## References

Jira ticket:
* https://getpocket.atlassian.net/browse/INFRA-568

### CODE REVIEW

**Review by kschelonka — COMMENTED** [review]
@kkyeboah thanks for the updates! I think I need someone else to approve since I started this pr

File: lambda/events/index.ts [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+    try {
+      const message = JSON.parse(JSON.parse(record.body).Message);
+      if (handlers[message['detail-type']] == null) {
+        throw new Error(
+          `Unable to retrieve handler for detail-type='${message['detail-type']}'`
+        );
+      }
+      await handlers[message['detail-type']](record);
+    } catch (error) {
+      console.log(error);
```
Comment (jpetto) [inline lambda/events/index.ts:22]: just noting the `console.log` here (these def help in debugging once in infrastructure - not saying to remove it)

**Review by jpetto — DISMISSED** [review]
code looks good to me. not a blocker, but i'd love to see some comments - maybe just business-logic related on the two lambda classes.

**Review by jpetto — APPROVED** [review]
comments!! 🚀

**Pull request merged**